apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -43,81 +43,75 @@ export function Lti13CourseNavigationInstructor({
             <button
               type="button"
               class="btn btn-sm btn-info"
-              data-toggle="modal"
-              data-target="#terminology-modal"
+              data-bs-toggle="modal"
+              data-bs-target="#terminology-modal"
             >
               New here? Learn about our terminology
             </button>
           </p>
 
-          <p>
-            If you want this to connect to a <strong>new course</strong> or
-            <strong>new course instance</strong>, you need to create those first and then return to
-            this form.
-          </p>
-          <ul>
-            <li>
-              To create a new course, please <a href="/pl/request_course">request one here</a>.
-            </li>
-            <li>
-              New course instances can be made from the "Course Instances" tab in an existing
-              course.
-            </li>
-            <ul>
-              <li>
-                If you want to copy an existing course instance, you can do that from its Course
-                Instance "Settings" tab.
-              </li>
-            </ul>
-          </ul>
-
           ${courses.length === 0
             ? html`<p>
-                It doesn't look like you have any PrairieLearn courses.
-                <a href="/pl/request_course" class="btn btn-success">Go request a course</a>
+                <strong>
+                  It doesn't look like you have course editor permissions in any PrairieLearn
+                  courses.
```

Comment:
IMO we should prefer more direct and definitive language:

```suggestion
                  You don't have course editor permissions in any PrairieLearn courses.
```

Reply:
Addressed in 69f0ba1